packages/rum/src/domain/record/serialize.ts

```diff
@@ -34,25 +35,33 @@ type ParentNodePrivacyLevel =
   | typeof NodePrivacyLevel.MASK
   | typeof NodePrivacyLevel.MASK_USER_INPUT
 
-export type SerializationContext = 'full-snapshot' | 'mutation'
+export enum SerializationContext {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏you could use a const enum here